app/templates/dashboard/_misc_index.html

```diff
@@ -60,43 +76,66 @@ <h3 class="heading-xmedium">Buyers</h3>
   <h2 class="heading-large">Outcomes</h2>
   <ul class="govuk-list">
     <li>
-      <a class="govuk-link" href="{{ url_for('.download_outcomes') }}">Download Direct Award outcomes</a>
+      <a class="govuk-link" href="{{ url_for('.download_outcomes') }}">
+        Download Direct Award outcomes
+      </a>
     </li>
   </ul>
 
   {# FRAMEWORKS AND APPLICATIONS #}
   <h2 class="heading-large">Manage applications</h2>
 
   {% for framework in frameworks %}
-    {% if framework.status in ["standstill", "live"] or (framework.family == 'digital-outcomes-and-specialists' and framework.status == 'expired') or current_user.has_any_role('admin-framework-manager', 'admin-ccs-sourcing') %}
+    {% if framework.status in ["standstill", "live"] or
+        (framework.family == 'digital-outcomes-and-specialists' and framework.status == 'expired') or
+          current_user.has_any_role('admin-framework-manager', 'admin-ccs-sourcing') %}
       <h3 class="heading-xmedium">{{framework.name}}</h3>
 
       <ul class="govuk-list">
-        {% if framework.status in ["live", "standstill"] or (framework.family == 'digital-outcomes-and-specialists' and framework.status == 'expired') %}
+        {% if framework.status in ["live", "standstill"] or
+            (framework.family == 'digital-outcomes-and-specialists' and framework.status == 'expired') %}
           {% set agreement_link_text = {
             'admin-ccs-category': 'View agreements',
             'admin-ccs-sourcing': 'Countersign agreements',
             'admin-framework-manager': 'View agreements',
             }
         %}
           <li>
-            <a class="govuk-link" href="{{ url_for('.list_agreements', framework_slug=framework['slug'], status='signed') }}">{{ agreement_link_text[current_user.role] }}</a>
+            <a class="govuk-link" href="{{ url_for('.list_agreements', framework_slug=framework['slug'], status='signed') }}">
+              {{ agreement_link_text[current_user.role] }}
+            </a>
           </li>
         {% endif %}
 
-        {% if current_user.has_any_role('admin-ccs-sourcing', 'admin-framework-manager') %}
+        {% if current_user.has_role('admin-ccs-sourcing') %}
           <li>
-            <a class="govuk-link" href="{{ url_for('public.view_statistics', framework_slug=framework['slug']) }}">View application statistics</a>
+            <a class="govuk-link" href="{{ url_for('public.view_statistics', framework_slug=framework['slug']) }}">
+              View application statistics
+            </a>
           </li>
-        {% endif %}
-        {% if current_user.has_role('admin-framework-manager') %}
```

Comment:
It's not clear to me what this change in logic is doing; it looks like we're just repeating the same HTML snippet twice now?

Reply:
It is repeating HTML, The reason for the change is to try to make it easier to read and understand what a specific type of user should be able to see by looking at one if statement rather than trying to build up a mental model of what it is they are meant to see on the page.

Does it make it easier to read?

Reply:
I would say it is easier to read. I'm not sure whether it's easier to maintain, I guess it depends on whether we expect the capabilities of roles to change a lot. Either layout is fine though.